content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -0,0 +1,656 @@
+
+```YML
+  actions:
+      type:   set
+        field:  description
+        locale: en_US
+        scope:  ecommerce
+        value:  "My very new description for purple tshirt"
+```
+
+## Remove
+
+The `Remove` action enables you to remove values from a multi-select attribute, a reference entity multiple link attribute or a product category.
+
+The expected values are:
+- `field`: the attribute code or “categories”
+- `items`: the values to remove
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+- `include_children`: if true, then it also applies the removal of the children to the given categories. It is only applicable if the field is set to “categories” (It's optional. By default, it is set to false)
+
+
+### Example
+
+To remove the “t-shirts” category, the action will be as follows:
+
+```YML
+  actions:
+      type: remove
+        field: categories
+        items:
+          - t-shirts
+```
+
+To remove the “clothing” category and its children, the action will be as follows:
+```YML
+  actions:
+      type: remove
+      field: categories
+      items:
+        - clothing
+      include_children: true
+```
+
+To unclassify products from the whole “Master catalog” tree, the action will be as follows:
+
+```YML
+  actions:
+      type: remove
+      field: categories
+      items:
+        - master
+      include_children: true
+```
+
+## Concatenate
+
+The `Concatenate` action concatenates at least two values into a single value. A space separates each source value.
+
+The possible source attribute types are:
+ - text
+ - text area
+ - date
+ - identifier
+ - metric
+ - number
+ - price collection
+ - simple or multi select (option codes)
+
+The possible target attribute types are:
+ - text
+ - text area
+
+**The parameters `from` and `to` are required in this format. Depending on the source attribute type, some optional keys can be added.**
+
+The expected values are:
+
+**`from`**
+- `field`: the attribute code.
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+- For `date` attribute: the format of the date following the PHP format [specification](https://www.php.net/manual/en/function.date.php) (Optional). By default, it is *Y-m-d* (e.g. *2020-01-31*)
+- For `price collection` attribute: the currency code for which the price is assigned (Optional). By default all the prices in the collection are displayed, separated by a coma.
+
+**`to`**
+- `field`: the attribute code.
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+
+### Example
+
+To concatenate the **brand** (non localizable and non scopable) and the **model** in the `en_US` locale into the **description** value in the `en_US` locale, the action will be as follows:
```

Comment:
Brand & model into a Product title would be a better match I guess. Unless you're targeting short description ;)